app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -42,6 +44,10 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        if (MvpFeatureManager.isEnabled) {
+            AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
```

Comment:
Ah, I see the answer I'm looking for in the message of the previous commit. Let's add a similar note here too.

> Because the app theme was changed to DayNight it automatically load dark
> colors for some attributes when the app is in light mode but for now we
>  don't want that.
> Those attributes will be updated at light theme implementation

Reply:
Considering the last commit I think I don't need this anymore, so can be removed. I will do some tests to check